bool A
bool B
bool X
bool Y
bool LB
bool RB
bool START
bool BACK
bool POWER
bool DPad_Up
bool DPad_Down
bool DPad_Left
bool DPad_Right
bool JoyLeft_PB
bool JoyRight_PB
 
